src/featurizations/weavefeaturization.jl

```diff
@@ -2,7 +2,7 @@
 
 struct WeaveFeaturization <: AbstractFeaturization
     atom_features::Vector{AtomFeature}
```

Comment:
this can become `Vector{<AbstractAtomFeature}` once I make the other hierarchy fixes

Reply:
Fixed in 7e53044.